dstep/translator/Record.d

```diff
@@ -105,6 +108,38 @@ void translateAnonymousRecord(Output output, Context context, Cursor cursor, Cur
         translateRecordDef(output, context, cursor, true);
 }
 
+bool shouldSkipRecord(Context context, Cursor cursor)
+{
+    if (cursor.kind == CXCursorKind.CXCursor_StructDecl ||
+        cursor.kind == CXCursorKind.CXCursor_UnionDecl)
+    {
+        auto typedefp = context.typedefParent(cursor.canonical);
+
+        if (typedefp.isValid && cursor.spelling == "")
+            return context.options.skipSymbols.contains(typedefp.spelling);
+        else
+            return context.options.skipSymbols.contains(cursor.spelling);
+    }
+
+    return false;
+}
+
+bool shouldSkipRecordDefinition(Context context, Cursor cursor)
+{
+    if (cursor.kind == CXCursorKind.CXCursor_StructDecl ||
+        cursor.kind == CXCursorKind.CXCursor_UnionDecl)
+    {
+        auto typedefp = context.typedefParent(cursor.canonical);
+
+        if (typedefp.isValid && cursor.spelling == "")
+            return context.options.skipDefinitions.contains(typedefp.spelling);
+        else
+            return context.options.skipDefinitions.contains(cursor.spelling);
```

Comment:
You can store the spelling in a local variable to avoid repeating the function call.